ext/frontend/src/app/auth/services/auth.service.ts

```diff
@@ -67,6 +67,21 @@ export class AuthService {
       );
   }
 
+  public authAgent(token: Token): Observable<{ agentToken: string }> {
+    const headers: HttpHeaders = new HttpHeaders({
```

Comment:
Tutaj te nagłowki potrzebne są? Wydaje mi sie że interceptor od autha by to ogarnął chyba. Mozna sprobowac to wywalic i zerknac czy przechodzi a jak nie no to zostawic

Reply:
yep it works